File: src/test/java/connectivity/JdsDbPostgreSqlmplementation.java
package connectivity;

import io.github.subiyacryolite.jds.JdsDbPostgreSql;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.util.Properties;

/**
 * @author indana
 */
public class JdsDbPostgreSqlmplementation extends JdsDbPostgreSql {

    @Override
    public Connection getConnection() throws ClassNotFoundException, SQLException {
        Class.forName("org.postgresql.Driver");
        Properties properties = new Properties();
        properties.put("user", "postgres");
        properties.put("password", "");
        return DriverManager.getConnection("jdbc:postgresql://127.0.0.1:5432/jds", properties);
    }
}

File: src/test/kotlin/connectivity/JdsDbPostgreSqlmplementation.kt
package connectivity

import io.github.subiyacryolite.jds.JdsDbPostgreSql
import java.io.File
import java.io.FileInputStream
import java.sql.Connection
import java.sql.DriverManager
import java.util.*

class JdsDbPostgreSqlmplementation : JdsDbPostgreSql() {

    private val properties: Properties = Properties()
    private val connectionString: String

    init {
        FileInputStream(File("db.pg.properties")).use { properties.load(it) }
        connectionString = "jdbc:postgresql://${properties["dbUrl"]}:${properties["dbPort"]}/${properties["dbName"]}"
    }

    override val connection: Connection
        get () {
            Class.forName("org.postgresql.Driver")
            return DriverManager.getConnection(connectionString, properties)
        }
}